src/CMakeLists.txt

```diff
@@ -112,8 +112,27 @@ if(NOT UMF_DISABLE_HWLOC)
         provider/provider_os_memory_windows.c)
 
     set(UMF_LIBS ${UMF_LIBS} ${LIBHWLOC_LIBRARIES})
+
+    add_optional_symbol(umfMemspaceCreateFromNumaArray)
```

Comment:
maybe a TODO for later, but we shouldn't be removing functions if hwloc is not available. These functions should simply return NOT_SUPPORTED.

Reply:
Yes, I agree - that would definitely be better.

Reply:
I added an issue for that https://github.com/oneapi-src/unified-memory-framework/issues/649